display_list/display_list_builder.cc

```diff
@@ -4,6 +4,8 @@
 
 #include "flutter/display_list/display_list_builder.h"
 
+#include "display_list_color_source.h"
```

Comment:
Nit: `flutter/display_list/display_list_color_source.h` for consistency with the rest of the includes?

Reply:
My IDE does this sometimes and I always miss it. Done.